### PR TITLE
chore(go): remove Go 1.15 and 1.16 images

### PR DESCRIPTION
**Pull request by quartzmo**

### CODE REVIEW

**Review by codyoss — APPROVED** [review]

File: go/go119/requirements.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+# by the following command:
 #
 #    pip-compile --generate-hashes requirements.in
 #
 argcomplete==2.0.0 \
     --hash=sha256:6372ad78c89d662035101418ae253668445b391755cfe94ea52f1b9d22425b20 \
     --hash=sha256:cffa11ea77999bb0dd27bb25ff6dc142a6796142f68d45b1a26b11f58724561e
     # via nox
-cachetools==5.2.0 \
+cachetools==5.2.0 \q
```
Comment (codyoss) [inline go/go119/requirements.txt:11]: This q looks incorrect here, please confirm

File: go/go119/base-tooling-requirements.txt [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
-tomli==2.0.1 \
-    --hash=sha256:939de3e7a6161af0c887ef91b7d41a53e7c5a1ca976325f429cb46ea9bc30ecc \
-    --hash=sha256:de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f
-wheel==0.37.1 \
-    --hash=sha256:4bdcd7d840138086126cd09254dc6195fb4fc6f01c050a1d7236f2630db1d22a \
-    --hash=sha256:e9a504e793efbca1b8e0e9cb979a249cf4a0a7b5b8c9e8b65a5e39d49529c1c4
-pip==22.2.2 \
-    --hash=sha256:3fd1929db052f056d7a998439176d3333fa1b3f6c1ad881de1885c0717608a4b \
-    --hash=sha256:b61a374b5bc40a6e982426aede40c9b5a08ff20e640f5b56977f4f91fed1e39a
+build==0.10.0     --hash=sha256:af266720050a66c893a6096a2f410989eeac74ff9a68ba194b3f6473e8e26171     --hash=sha256:d5b71264afdb5951d6704482aac78de887c80691c52b88a9ad195983ca2c9269
```
Comment (codyoss) [inline go/go119/base-tooling-requirements.txt:1]: Is this file formatted correctly?

File: go/go119/requirements.txt (hunk 1 above)
Comment (quartzmo) [reply go/go119/requirements.txt]: Fixed, thank you!

File: go/go119/base-tooling-requirements.txt (hunk 2 above)
Comment (quartzmo) [reply go/go119/base-tooling-requirements.txt:1]: Apparently so, it works. Compact is the new hotness?

**Review by codyoss — APPROVED** [review]

**Pull request merged**